client/src/components/post/Posts.js

```diff
@@ -14,30 +15,40 @@ const Posts = () => {
   const postContext = useContext(PostContext)
   const { posts } = postContext
 
+  const authContext = useContext(AuthContext)
+  const { loggedInUser } = authContext
+
   return (
     <div>
       {posts.map((post) => (
         <div className='card text-dark bg-light mb-3' key={post.id}>
-          <div className='card-header d-flex justify-content-between'>
-            <div className='d-flex align-items-center '>
-              <img
-                src={profilePicture}
-                alt='profile'
-                className='post-profile-picture border border-secondary rounded mr-3'
-                id='profile-picture'
-              />
-              <Link to={`/user/${post.user_id}`}>
-                <h6 className='text-dark'>
-                  {post.first_name} {post.last_name}
-                </h6>
-              </Link>
-            </div>
-            <div className='d-flex align-items-end'>
-              {convertDate(post.created_at)}
+          <div className='card-header'>
+            <div className='d-flex justify-content-between'>
+              <div className='d-flex align-items-center '>
+                <img
+                  src={profilePicture}
```

Comment:
small nit, but what do you think about clarifying this variable by calling it `profilePictureLink` or `profilePictureSrc` or something of that nature. i could go with or without this change, just a thought